### PR TITLE
Add automated troubleshooting documentation

### PR DESCRIPTION
**Pull request by dfarrell07**
Document the standard process to use subctl to debug Submariner issues
and collect information for getting help.

This is becoming something we answer a lot, so it's good to have a
pointer with a prepared answer.

Signed-off-by: Daniel Farrell <dfarrell@redhat.com>

### CODE REVIEW

**Review by nyechiel — APPROVED** [review]

**Comment by dfarrell07** [comment]
I updated this to also mention the bug report template, as you added on Slack @nyechiel.

**Review by tpantelis — APPROVED** [review]

File: src/content/operations/troubleshooting/_index.en.md [hunk 1]
```diff
@@ -15,6 +15,40 @@ Basic familiarity with the Submariner components and architecture will be helpfu
...
+Make sure `KUBECONFIG` is set to point at your connected clusters:
+
+```shell
+export KUBECONFIG=<paths>
+```
+
+Get an idea of what's failing:
+
+```shell
+subctl diagnose all
```
Comment (sridhargaddam) [inline src/content/operations/troubleshooting/_index.en.md:37]: Thanks for the PR @dfarrell07. Can you also include the following command.
`subctl diagnose firewall inter-cluster`

Comment (dfarrell07) [reply src/content/operations/troubleshooting/_index.en.md:37]: Added, thanks @sridhargaddam!

**Comment by dfarrell07** [comment]
IMHO, let's wait 24h before merging to see if anyone else has suggestions.

File: src/content/operations/troubleshooting/_index.en.md (hunk 1 above)
Comment (sridhargaddam) [reply src/content/operations/troubleshooting/_index.en.md:37]: Thanks @dfarrell07

**Review by sridhargaddam — APPROVED** [review]

**Review by maayanf24 — APPROVED** [review]

**Pull request merged**